client/coral-framework/components/Slot.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {getSlotElements} from 'coral-framework/helpers/plugins';
+
+class Slot extends Component {
+  render() {
+    const {fill, ...rest} = this.props;
+    return (
+      <div>
+        {getSlotElements(fill, rest)}
+      </div>
+    );
+  }
+}
+
```

Comment:
could we get some `PropTypes` here? Also, this component would be a great place to add a ton of comments to help guide contributors trying to create their own plugin.